include/crm/common/remote_internal.h

```diff
@@ -89,9 +89,13 @@ int pcmk__read_handshake_data(const pcmk__client_t *client);
  *
  * \param[in,out] remote       Newly established remote connection
  * \param[in]     timeout_sec  Abort handshake if not completed within this time
+ * \param[out]    gnutls_rc    If this is non-NULL and this function returns
+ *                             EPROTO, the GnuTLS rc will be stored here (for
+ *                             logging)
  *
```

Comment:
maybe worth mentioning that it will be updated to SUCCESS in case of !EPROTO

Reply:
Well there are error-cases where it isn't updated so just valid if EPROTO but should we even touch it then otherwise or update to some generic error (haven't found a good one though) for other return codes ...

Reply:
It will always be SUCCESS for anything other than EPROTO. I can mention that

Reply:
updated commit